src/ManojN/Q2arrayDuplicateNumber.java

```diff
@@ -0,0 +1,24 @@
+//Q2) Find duplicate number on Integer array in Java?
+
+package ManojN;
+
+public class Q2arrayDuplicateNumber {
+
+	static void m1(int a2[]){
+		//int a2 [] = {0, 3, 1, 2, 3};
```

Comment:
Hi @ManojNAug19 
Here check output for
int a2[] = { 0, 3, 1, 2, 3,3 };

it print  
missing no in array : 3
missing no in array : 3
missing no in array : 3

Start debugging....